CSV/specifications.csv

```diff
@@ -91,9 +91,11 @@ QT,QT,"The <a href='http://www.apple.com/quicktime/' target='_blank'>QuickTime</
 RealHD,RealHD,<a href='http://www.realnetworks.com/realmediahd'>RealMedia™ HD</a>
 ross,Ross,<a href='http://www.rossvideo.com/'>Ross Video</a>
 samsung,Samsung,"<a href='http://www.samsung.com/'>Samsung</a> "
+SCTE, SCTE 215-1-1 2020b,"<a href='https://www.scte.org/download-scteisbe-standards/'>SCTE Specifications</a>"
```

Comment:
The first column is the link name for the specification. It is used in the last column of the brands.csv file. You currently have `SCTE` vs `SCTE 215-1-1b`. You should pick one and we don't allow spaces in link names. Use `SCTE-215-1-1b` in both files.